### ISSUE TITLE
[Create/edit survey] Warn user if no LOIs and "free form" mode disabled - warn that survey will be unusable

### ISSUE THREAD

**Issue by gino-m** [google/ground-platform#1667]

**Comment by gino-m** [comment]
@lecrabe FYI

**Comment by amegantz** [comment]
![image](https://github.com/user-attachments/assets/c0dbcd96-8c0f-4bf2-b16b-7b6106ce7fbd)

No data collection sites
Import predefined sites or enable free-form data collection to continue.
[Go back]

I kinda like the idea of disabling the "Continue" button and including the content above somewhere near the disabled button, but I guess a dialog works too.

**Comment by rfontanarosa** [comment]
@amegantz yes, no problem to me to open a popup instead of disable the continue button. we can add this behaviour here: https://github.com/google/ground-platform/issues/1620 (I'm working on it right now). Just let me know what you prefer. @gino-m wdyt?

**Comment by gino-m** [comment]
@kenstershiro FYI
